packages/core/src/graphql/index.ts

```diff
@@ -564,22 +566,33 @@ async function executePluralQuery(
   const after = args.after ?? null;
   const before = args.before ?? null;
 
+  if (after !== null && before !== null) {
+    throw new Error("Cannot specify both before and after cursors.");
+  }
+
   let startCursor = null;
   let endCursor = null;
   let hasPreviousPage = false;
   let hasNextPage = false;
 
-  if (after !== null && before !== null) {
-    throw new Error("Cannot specify both before and after cursors.");
-  }
+  const totalCountPromise = includeTotalCount
+    ? drizzle
+        .select({ count: count() })
+        .from(rawTable)
+        .where(and(...whereConditions, ...extraConditions))
+        .then((rows) => rows[0]?.count ?? null)
+    : Promise.resolve(null);
```

Comment:
Does this mean that queries that don't request total count will return data with `totalCount: null` ?

Reply:
Nope - GraphQL will ignore/strip any unselected properties on the object returned by the resolver.